lib/puppet/provider/fullchain/posix.rb

```diff
@@ -0,0 +1,62 @@
+require 'pp'
```

Comment:
I'm guessing this is left behind from local debug.

Reply:
No, Ruby's OpenSSL library does have a [pretty_print](https://github.com/ruby/openssl/blob/74731d9406eb80095bdc694b18eaa0cecd2730aa/lib/openssl/x509.rb#L331) feature which I make use of, e.g. at https://github.com/rtib/puppet-openssl/blob/539fdd2f48ab369cdc82364dba3bcba1ab1528b6/lib/puppet/provider/fullchain/posix.rb#L18 for output messages. It won't make sense to output log entries with a certificate in any other representation. Therefore the pretty print library is needed here.